elide-datastore/elide-datastore-aggregation/pom.xml

```diff
@@ -59,6 +59,22 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
```

Comment:
`<scope>test</scope>` dependencies should be placed after regular compile dependencies.